GPyOpt/core/bo.py

```diff
@@ -13,7 +13,7 @@
 from ..core.task.cost import CostModel
 from ..optimization.acquisition_optimizer import ContextManager
 try:
-    from ..plotting.plots_bo import plot_acquisition, plot_convergence
+    from GPyOpt.plotting.plots_bo import plot_acquisition, plot_convergence
 except:
     pass
```

Comment:
Please add a warning message when the import fails.

Reply:
I added a warning message.

Reply:
Am i (not) seeing things, or this change is no longer there?